src/java.compiler/share/classes/javax/tools/StandardJavaFileManager.java

```diff
@@ -267,7 +267,8 @@
      * @throws IllegalArgumentException if the array of files includes
      * a directory
```

Comment:
consider adding ` or if this file manager does not support any of the
     * given paths` in line with similar methods